string desired_configuration # configuration scatter, stack_ascending, stack_descending
int32 target_block
